Makefile

```diff
@@ -489,22 +477,37 @@ $(HELM): | $(LOCALBIN)
 	curl -s --fail $(HELM_INSTALL_SCRIPT) | USE_SUDO=false HELM_INSTALL_DIR=$(LOCALBIN) DESIRED_VERSION=$(HELM_VERSION) BINARY_NAME=helm-$(HELM_VERSION) PATH="$(LOCALBIN):$(PATH)" bash
 
 $(FLUX_HELM_CRD): | $(EXTERNAL_CRD_DIR)
+	FLUX_HELM_VERSION ?= $(shell go mod edit -json | $(JQ) -r '.Require[] | select(.Path == "github.com/fluxcd/helm-controller/api") | .Version')
+	FLUX_HELM_NAME ?= helm
+	FLUX_HELM_CRD ?= $(EXTERNAL_CRD_DIR)/$(FLUX_HELM_NAME)-$(FLUX_HELM_VERSION).yaml
```

Comment:
I don't think it's justified here, since:
1. Syntax looks incorrect
2. You introduced duplication
3. You didn't add `jq` to the target requirements (I assume that was the idea)

I suggest you just change `jq` -> `$(JQ)` in the global definitions and leave it as it were.

Reply:
Hey @a13x5, I've descoped the PR to address `jq dependecy` and `unzip verfication` to unblock the PR. 
I might raise another PR in future once I find a way to remove the duplication while addressing the original issue.